packages/ketcher-react/src/script/ui/component/form/Select/Select.module.less

```diff
@@ -20,6 +20,7 @@
 // stylelint-disable no-descending-specificity
 .dropdownList {
   :global(.MuiPaper-root) {
+    max-height: 200px;
```

Comment:
I think we should use the same scroll which we have in Templates and Settings windows